Makefile

```diff
@@ -0,0 +1,144 @@
+# From https://raw.githubusercontent.com/vincentbernat/hellogopher/1cf9ebeeaea41d2e159e196f7d176d6b12ff0eb2/Makefile
+# Licensed under Creative Commons Zero v1.0 Universal (CC0)
+
+MODULE   = $(shell env GO111MODULE=on $(GO) list -m)
+DATE    ?= $(shell date +%FT%T%z)
+VERSION ?= $(shell git describe --tags --always --dirty --match=v* 2> /dev/null || \
+			cat .version 2> /dev/null || echo v0)
+PKGS     = $(or $(PKG),$(shell env GO111MODULE=on $(GO) list ./...))
+BIN      = bin
+
+GO      = go
+TIMEOUT = 15
+V = 0
+Q = $(if $(filter 1,$V),,@)
+M = $(shell [ "$$(tput colors 2> /dev/null || echo 0)" -ge 8 ] && printf "\033[34;1m▶\033[0m" || printf "▶")
+
+export GO111MODULE=on
+
+.PHONY: all
+all: check-license mocks deep-copy fmt lint test-coverage | $(BIN)
```

Comment:
Note: running the `all` target right now will bring up hundreds of lint problems, most of them related to exported types without comments. I did not fix those in this PR. I plan to do it later on.